docs/DeveloperGuide.md

```diff
@@ -294,16 +298,16 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is the `NurseyBook` and the **Actor** is the `user`, unless specified otherwise)
 
-**Use case: Delete a person**
+**Use case: Delete a task**
 
 **MSS**
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1.  User requests to list tasks
+2.  NurseyBook shows a list of tasks
+3.  User requests to delete a specific task in the list
+4.  NurseyBook deletes the person
```

Comment:
Instead of 'NurseyBook deletes the person', should it be 'NurseyBook deletes the task' instead?